File: distribution/RealmIntroExample/app/src/main/java/io/realm/realmintroexample/Dog.java
package io.realm.realmintroexample;

import io.realm.RealmObject;
import io.realm.annotations.RealmClass;

@RealmClass
public class Dog extends RealmObject {
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: examples/coroutinesExample/src/main/java/io/realm/examples/coroutinesexample/data/local/model/Dog.kt
package io.realm.examples.coroutinesexample.data.local.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Dog : RealmObject() {
    @PrimaryKey
    var name: String = ""
    var age: Int = 1
    var owner: String = ""      // We have an owner but we won't show it on screen

    override fun toString(): String = "Name: $name, age: $age, owner: $owner"
}
